docs/development/CUSTOM_RULES.md

```diff
@@ -306,7 +306,7 @@ const ruleTester = new ESLintUtils.RuleTester({
   }
 });
 
-ruleTester.run('my-typed-rule', rule {
+ruleTester.run('my-typed-rule', rule, {
```

Comment:
@GalenWong the [build failure](https://github.com/typescript-eslint/typescript-eslint/runs/4565717340?check_suite_focus=true) indicates you'll need to run Prettier on this file.

Reply:
I ran the prettier and it introduced some other changes. I guess the prettier only looks through touched files? That's how other PR can be built despite the formatting issue was always there?